BOUNTY.md

```diff
@@ -0,0 +1,44 @@
+# Bounty
+
+This repository is part of the Open Collective bounty program. Get paid contributing to Open Source!
+
+## Intro
+
+The Open Collective engineering team is small and we're always looking for new contributors to our Open Source codebases. Our Bounty program is an opportunity to solve issues that are currently difficult to prioritize and will likely be negligated otherwise. Contributors doing the effort to work on these issues will be rewarded in cash or gift cards.
+
+Our Bounty model is simple:
+
+- 100\$: for simple or unknown complexity
+- 300\$: for medium complexity
+- 1000\$: for high complexity
+
+We want to attract Open Source contributors willing to make the world a better place, money is nice but should ideally not be the primary motivation. There should be no compromise on quality, the issue will only be considered completed when the Pull Request is merged by a Core Contributor.
+
+## How it works?
+
+### Workflow for Bounty Hunters
+
+1. Search for Issues with attached bounties in our main repository:
+
+- [Bounties for all repositories](https://github.com/opencollective/opencollective/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Abounty)
+- [Bounties for this repository](https://github.com/opencollective/opencollective/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Abounty+label%3Afrontend) (opencollective-frontend):
+
+1. Express interest by commenting on the issue and ask to be assigned.
+
+1. Open a Pull Request and ask for feedback/review
+
+1. If applicable, implement feedback from Core Contributors
+
+1. PR is approved and gets merged by a Core Contributor
+
+1. Get paid:
+   - If you can issue an invoice, submit it as expense to [Open Collective Engineering](https://opencollective.com/engineering).
+   - If you can't issue an invoice, ask to be rewarded with an Open Collective Gift Card.
+
+### Workflow for Core Contributors
+
+1. Tag issues with the "bounty" label and another tag with the bounty amount (ie: "100\$")
+
+1. Add a comment with basic explanations of the Bounty process and a link to BOUNTY.md
```

Comment:
It would be nice to provide a template for the comment to add on the issue

Reply:
That was integrated.